src/spatial/utils.jl

```diff
@@ -0,0 +1,219 @@
+"""
+A file with types, structs and functions for spatial simulations
+"""
+
+"""
+stores info for a spatial jump
+"""
+struct SpatialJump{J}
```

Comment:
Maybe

```julia
struct SpatialJump{J}
    src::J    # source location
    jidx::Int # index of jump as a diffusion hop or reaction
    dst::J    # destination location
end
```

Reply:
changed

Reply:
Why is the location parameterized?

Reply:
In case we ever want to use something like `CartesianIndex` for site locations instead of integers